src/pages/faq.tsx

```diff
@@ -0,0 +1,64 @@
+import { Collapse, Text } from '@geist-ui/core';
+import { useRouter } from 'next/router';
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown';
+
+const FAQ = () => {
+  const FAQ = [
+    {
+      Name: 'AvdanOS',
+      Questions: [
+        {
+          QuestionNumber: 1,
+          Title: 'What will AvdanOS be?',
+          SubTitle: '',
```

Comment:
Maybe consider using null instead of empty string. Empty string may cause unintended behavior like rendering of extra gap.

Reply:
Okay